pkg/util/tmplvar/parse.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package tmplvar
```

Comment:
All this code was pulled out of `autodiscovery` so it can be used by the `tagger` too.

Reply:
not a fan of `tmplvar` package name

Reply:
I didn't want it to because of stdlib `text/template` and this package is only responsible for the logic of parsing template variables.

Reply:
Right forgot about `text/template`, I guess that's fair enough if we can't find a better naming 👌